cypress/integration/features/teams/createEditDeleteTeam.feature

```diff
@@ -0,0 +1,41 @@
+Feature: CRUD operation on Teams
```

Comment:
Should be like for better readability in html reports 
`Feature: Team - CRUD operation on Teams`

Reply:
updated it.